routes.py

```diff
@@ -15,11 +21,21 @@
 
 @app.route("/")
 def index():
+    """
+    This route displays an HTML frontend Hello World
+
+    :return: Rendered HTML template
+    """
     return render_template('index.html', who="World")
 
 
 @app.route("/rand_num")
 def rand_num():
+    """
+    This route displays an HTML frontend random number
+
+    :return: Rendered HTML template
+    """
```

Comment:
I love a good docstring. Is it the norm for them to be multiline, or would a single line be fine if the message was short enough?

Reply:
Yeah I think that would be cool. I'd love to see how to do that as well. I love that level of quality control in a group setting.